packages/api-explorer/src/form-components/SchemaField.jsx

```diff
@@ -28,6 +47,8 @@ function getCustomType(schema) {
 }
 
 function SchemaField(props) {
+  if (!doesFormatExist(props.schema.type, props.schema.format)) props.schema.format = undefined;
```

Comment:
Can you add a test for this case? Tests are written via this file: https://github.com/readmeio/api-explorer/blob/master/packages/api-explorer/__tests__/Params.test.jsx#L50-L63